src/Packlist/PacklistForm.js

```diff
@@ -164,6 +176,13 @@ export default function PacklistForm() {
     } else setItemError(true)
   }
 
+  function addMultiplyItems(templateName) {
+    const templatePacklist = packlists.find(
+      (packlist) => templateName.name === packlist.name
+    )
+    setItems([...templatePacklist.packlist, ...items])
+  }
+
   function deleteItem(index) {
     setItems([...items.slice(0, index), ...items.slice(index + 1)])
   }
```

Comment:
This could also be in the same costum hook. Something like useItems. Maybe you already have one, where you can put this.